### PR TITLE
Fixes #15166: Improve and correct tests for command_execution GM

### PR DESCRIPTION
**Pull request by Fdall**
https://issues.rudder.io/issues/15166

### CODE REVIEW

**Comment by Fdall** [comment]
PR updated with a new commit

**Comment by Fdall** [comment]
PR rebased

File: tests/acceptance/30_generic_methods/command_execution.cf [hunk 1]
```diff
@@ -22,26 +22,66 @@ body common control
...
-    "ok" expression => "command_execution_${init.canonified_command_name}_ok.!command_execution_${init.canonified_command_name}_error";
+    "file_1" expression => fileexists("${init.tmp}/1");
+    "file_2"        not => fileexists("${init.tmp}/my_file");
+    "classes_ok" expression => "ph1_ok.ph2_ok.ph3_ok.ph4_ok.ph5_ok.ph6_ok";
+
+    "ok" expression => "classes_ok.file_1.file_2";
 
   reports:
+    file_exists::
+      "le fichier ${init.tmp}/my_file existe";
```
Comment (amousset) [inline tests/acceptance/30_generic_methods/command_execution.cf]: better en english

**Comment by Fdall** [comment]
PR updated with a new commit

**Comment by Fdall** [comment]
OK, squash merging this PR

**Pull request merged**